dingo/bindings/bindings.cpp

```diff
@@ -131,20 +131,62 @@ double HPolytopeCPP::apply_sampling(int walk_len,
    } else if (method == 7) { // vaidya walk
       uniform_sampling<VaidyaWalk>(rand_points, HP, rng, walk_len, number_of_points,
                                    starting_point, number_of_points_to_burn);
-   } else if (method == 8) { // gaussian sampling with gaussian HMC exact walk
+   } else if (method == 8) { // Gaussian sampling with exact HMC walk
       NT a = NT(1)/(NT(2)*variance);
       gaussian_sampling<GaussianHamiltonianMonteCarloExactWalk>(rand_points, HP, rng, walk_len, number_of_points, a,
                                    starting_point, number_of_points_to_burn);
-   } else if (method == 9) { // exponential sampling with exponential HMC exact walk
+   } else if (method == 9) { // exponential sampling with exact HMC walk
       VT c(d);
       for (int i = 0; i < d; i++){
          c(i) = bias_vector_[i];
       }
       Point bias_vector(c);       
       exponential_sampling<ExponentialHamiltonianMonteCarloExactWalk>(rand_points, HP, rng, walk_len, number_of_points, bias_vector, variance,
                                    starting_point, number_of_points_to_burn);
-      } 
-   
+   } else if (method == 10) { // HMC with Gaussian distribution
```

Comment:
plz move the code inside this `else if () {...}` to a new function in a header file, so we have more clean and well structured code.